### PR TITLE
refactor: move ginkgo wrapper related methods out of util

### PR DESCRIPTION
**Pull request by MorrisLaw**
**What type of PR is this?**
/kind cleanup
/priority backlog

**What this PR does / why we need it**:
[#77095](https://github.com/kubernetes/kubernetes/issues/77095)

Moves `wrapper` related methods out of `util` and into an individual `wrappers.go` file, inside of `test/e2e/framework/`

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by andrewsykim** [comment]
Thanks for the PR @MorrisLaw! 

I think a better name for `wrappers.go` might be `ginkgo_wrappers.go`. The intention there was to move all the Skip* and Fail* functions that were just wrappers for ginkgo there. The Kubectl and wait functions belong in other packages IMO. What do you think?

**Comment by MorrisLaw** [comment]
Yeah that makes sense @andrewsykim. So if I make a `ginkgo_wrappers.go` file, should they all be consolidated here? Or should I move these skip and fail, ginkgo related methods, to the already existing `test/e2e/framework/gingkowrapper/wrapper.go` file? It looks like it's just for annotating panics, but it seems redundant to have a `gingkowrapper.go` file next to a `ginkgowrapper` package.

**Comment by andrewsykim** [comment]
@MorrisLaw good catch, I didn't realize we had a ginkgowrapper package. 

I'm leaning towards consolidating everything under `test/e2e/framework/ginkgo_wrappers.go` because there are way more callers to the `framework` package as opposed to `ginkgowrapper` and most of the calls to `ginkgowrapper` are from `test/e2e/framework` already. What do you think?

**Comment by MorrisLaw** [comment]
I agree. Consolidating into one file would be a cleaner approach. I'll update the PR @andrewsykim

**Comment by MorrisLaw** [comment]
Just pushed the update, what do you think? @andrewsykim

File: test/e2e/framework/ginkgo_wrappers.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```
Comment (andrewsykim) [inline test/e2e/framework/ginkgo_wrappers.go]: The copy right date shouldn't change if we are moving a file

**Review by andrewsykim — COMMENTED** [review]
Thanks @MorrisLaw! Can we also move `Failf` and all the `Skip*` functions to ginkgo_wrappers.go?

**Comment by MorrisLaw** [comment]
@andrewsykim all of the `Failf` and `Skip*` have been moved to `ginkgo_wrappers`, nice catch! Should I also move `RunIfContainerRuntimeIs` and `RunIfSystemSpecNameIs` since they call `skipInternalf` as well?

File: test/e2e/framework/ginkgo_wrappers.go [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+		if e != nil {
+			panic(sp)
+		}
+	}()
+
+	ginkgo.Skip(message, skip)
+}
+
+// Skipf skips with information about why the test is being skipped.
+func Skipf(format string, args ...interface{}) {
```
Comment (WanLinghao) [inline test/e2e/framework/ginkgo_wrappers.go]: It seems you didn't delete the duplicate definition in util.go?

**Review by timothysc — CHANGES_REQUESTED** [review]
needs more shuffling from comment.

File: test/e2e/framework/ginkgo_wrappers.go (hunk 2 above)
Comment (MorrisLaw) [reply test/e2e/framework/ginkgo_wrappers.go]: Nice catch @WanLinghao ! I'll update this in a bit

File: test/e2e/framework/util.go [hunk 3]
```diff
@@ -75,7 +74,12 @@ import (
 	"k8s.io/kubernetes/pkg/client/conditions"
 	"k8s.io/kubernetes/pkg/controller"
 	"k8s.io/kubernetes/pkg/controller/service"
+<<<<<<< HEAD
```
Comment (timothysc) [inline test/e2e/framework/util.go]: uhh, diff fail.

**Review by timothysc — COMMENTED** [review]
So in thinking about it, I'm ok with ginkgo wrappers living in the main framework package and not pushing them out.  In part b/c that is it's primary job.  Importing framework should give you the basic tools you need and if you need specific wrappers based on behavior you can import them with minimal dependance. 

/hold

**Comment by timothysc** [comment]
I'm going to close this specific PR b/c of the last comment.

**Pull request closed without merging**